explainaboard/tests/test_ner.py

```diff
@@ -95,59 +95,37 @@ def test_datalab_loader(self):
         self.assertEqual(n_buckets, 3)
 
         # 3. Unittest: test detailed bucket information: bucket interval
-        second_interval = list(sys_info.results.fine_grained["span_econ"].keys())[
-            1
-        ]  # [0.007462686567164179,0.9565217391304348]
+        # [0.007462686567164179,0.9565217391304348]
+        second_bucket = sys_info.results.fine_grained["span_econ"][1]
         self.assertAlmostEqual(
-            second_interval[0],
+            second_bucket.bucket_interval[0],
             0.007462686567164179,
             4,
             "almost equal",
         )
         self.assertAlmostEqual(
-            second_interval[1],
-            0.8603351955307262,
+            second_bucket.bucket_interval[1],
```

Comment:
it seems that the number value is a little different from the one in the previous version

Reply:
Yes, the reason for this is because previously bucketing was performed separately on true and predicted spans, but now it is performed on the union of both. I'm pretty sure it's not a bug, but the buckets will be a little bit different.